core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -310,8 +302,9 @@ class TransactionStateManager(brokerId: Int,
 
   // TODO: check broker message format and error if < V2
   def appendTransactionToLog(transactionalId: String,
+                             coordinatorEpoch: Int,
```

Comment:
Should we change this to have `CoordinatorEpochAndTxnMetadata` as a param rather than the 2 separate params?